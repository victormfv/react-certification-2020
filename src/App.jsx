import React from 'react';
import { BrowserRouter, Switch, Route } from 'react-router-dom';
import ReactDemo from './components/reactdemo';
import Home from './pages/Home';
import Layout from './components/Layout';
import NotFound from './pages/NotFound';
import VideosProvider from './state/VideosProvider';

function App() {
  return (
    <BrowserRouter>
      <VideosProvider>
        <Layout>
          <Switch>
            <Route exact path="/">
              <Home />
            </Route>
            <Route exact path="/react">
              <ReactDemo />
            </Route>
            <Route path="*">
              <NotFound />
            </Route>
          </Switch>
        </Layout>
      </VideosProvider>
    </BrowserRouter>
  );
}

export default App;
