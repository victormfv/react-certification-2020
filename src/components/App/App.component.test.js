import React from 'react';
import renderer from 'react-test-renderer';
import App from './App.component';
describe('App Component', () => {
    it('renders correctly', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
    });
});