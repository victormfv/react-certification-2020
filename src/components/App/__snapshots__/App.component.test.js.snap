// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders correctly 1`] = `
<div>
  <div>
    <div
      className="sc-jrAGrp gROdRU"
    >
      <div
        className="sc-kEjbxe gDCpRM"
      >
        <div
          className="sc-iqHYGH nrqOM"
        >
          <a
            className="sc-kstrdz fLkyjB"
            href="#"
          >
            <svg
              aria-hidden="true"
              className="sc-pFZIQ bUaPrX"
              fill="currentColor"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
          </a>
          <div
            className="sc-crrsfI eZpzax"
          >
            <svg
              className="sc-fubCfw iXLixq"
              fill="none"
              height="24"
              width="24"
            >
              <path
                d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
                stroke="currentColor"
                strokeLinecap="round"
                strokeLinejoin="round"
                strokeWidth="2"
              />
            </svg>
            <input
              className="sc-dQppl fWIQTr"
              placeholder="Search"
              type="text"
            />
          </div>
        </div>
        <div
          className="sc-bqyKva jBclnG"
        >
          <a
            className="sc-kstrdz fLkyjB"
            href="#"
          >
            <div>
              <label
                className="sc-jSgupP hfyfcK"
              >
                <input
                  defaultChecked={true}
                  type="checkbox"
                />
                <span
                  className="sc-gKsewC fGTbrx"
                />
              </label>
               
              Dark Mode
            </div>
          </a>
          <a
            className="sc-kstrdz fLkyjB"
            href="#"
          >
            <svg
              aria-hidden="true"
              className="sc-iBPRYJ keFouh"
              fill="currentColor"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
              />
            </svg>
          </a>
        </div>
      </div>
    </div>
  </div>
  <main
    className="container"
  >
    <section
      className="homepage"
    >
      <h2>
        Welcome to the Challenge!
      </h2>
      <div
        className="sc-eCssSg iUvlzN"
      >
        
      </div>
    </section>
  </main>
</div>
`;
